tests/zkevm/test_worst_compute.py

```diff
@@ -1066,3 +1066,43 @@ def _generate_bn128_pairs(n: int, seed: int = 0):
         calldata = Bytes(calldata + pair_calldata)
 
     return calldata
+
+
+@pytest.mark.parametrize(
+    "calldata",
+    [
+        pytest.param(b"", id="empty"),
+        pytest.param(b"\x00", id="zero-loop"),
+        pytest.param(b"\x00" * 31 + b"\x20", id="one-loop"),
+    ],
+)
```

Comment:
Should this test not be marked `valid_from` since it uses PUSH0? (Shanghai)

This will OOG before Shanghai but it will thus not be seen directly from test that it is failing (but yes, zkEVM will report a super low amount of cycles!!)